src/main/java/com/puppycrawl/tools/checkstyle/checks/naming/AbbreviationAsWordInNameCheck.java

```diff
@@ -297,6 +297,38 @@
  * }
  * </pre>
  * <p>
+ * To configure to check variables, enforce
+ * no abbreviations (essentially camel case) except for
+ * words like 'ORDERS', 'OBSERVATION', 'UNDERSCORE', 'TEST'.
+ * </p>
+ * <p>Configuration:</p>
+ * <pre>
+ * &lt;module name="AbbreviationAsWordInName"&gt;
+ *     &lt;property name="allowedAbbreviations" value="ORDER, OBSERVATION, UNDERSCORE, TEST"/&gt;
+ * &lt;/module&gt;
+ * </pre>
+ * <p>Example:</p>
+ * <pre>
+ * public class InputAbbreviationAsWordInNameType7 {
+ *    void getTEST() {
+ *    } // OK
+ *
+ *    void getORDER_OBSERVATION() {} // OK
```

Comment:
Please remove All examples below this line

Reply:
Done

Reply:
Not done

Reply:
Is it ok, now ?